examples/easy_start/nipype_preproc_spm_auditory.py

```diff
@@ -48,12 +53,12 @@
 frametimes = np.linspace(0, (nscans - 1) * tr, nscans)
 drift_model = 'Cosine'
 hrf_model = 'Canonical With Derivative'
-design_matrix = make_dmtx(frametimes,
-                          paradigm, hrf_model=hrf_model,
-                          drift_model=drift_model, hfcut=hfcut)
+design_matrix = make_design_matrix(
+    frametimes, paradigm, hrf_model=hrf_model, drift_model=drift_model,
+    period_cut=hfcut)
 
 # plot and save design matrix
-ax = design_matrix.show()
+ax = plot_design_matrix(design_matrix)
 ax.set_position([.05, .25, .9, .65])
```

Comment:
At this point, the user is over the pen with the code complexity.This line looks like magic. Users shouldn't have to do such manual tweaking.